pom.xml

```diff
@@ -67,7 +67,8 @@
         <version.com.fasterxml.jackson.dataformat>2.10.0.pr1</version.com.fasterxml.jackson.dataformat>
         <version.commons-io>2.11.0</version.commons-io>
         <version.junit>4.13.2</version.junit>
-        <version.org.jsweet>2.3.5</version.org.jsweet>
+        <version.org.jsweet>2.3.10</version.org.jsweet>
```

Comment:
This will break the library when used in a browser for reasons I cannot determine.  See this GH issue: https://github.com/cincheo/jsweet/issues/628

Did you actually need to upgrade the jsweet version to get this working?

Reply:
No I did not, in fact I have already restored the original version, just have not pushed it yet.